Tuist/ProjectDescriptionHelpers/TargetAction+Initializing.swift

```diff
@@ -32,4 +32,14 @@ extension TargetAction {
             basedOnDependencyAnalysis: true
         )
     }
+
+    public static func swiftFormatAction() -> TargetAction {
+        let swiftFormatPath = "BuildTools"
+        let runSwiftFormat = "SDKROOT=(xcrun --sdk macosx --show-sdk-path)\n#swift package update #Uncomment this line temporarily to update the version used to the latest matching your BuildTools/Package.swift file\nswift run -c release swiftformat \"$SRCROOT\""
+        return .pre(
+            script: "cd \(swiftFormatPath)\n\(runSwiftFormat)",
+            name: "SwiftFormat",
+            basedOnDependencyAnalysis: true
+        )
```

Comment:
Can we use multiline string literal?
```suggestion
        let swiftFormatPath = "BuildTools"
        let runSwiftFormat = """
        SDKROOT=(xcrun --sdk macosx --show-sdk-path)
        #swift package update #Uncomment this line temporarily to update the version used to the latest matching your BuildTools/Package.swift file
        swift run -c release swiftformat \"$SRCROOT\"
        """
        let script = """
        cd \(swiftFormatPath)
        \(runSwiftFormat)
        """
        return .pre(
            script: script,
            name: "SwiftFormat",
            basedOnDependencyAnalysis: true
        )
```